panels/dev-state/ha-panel-dev-state.html

```diff
@@ -122,7 +122,7 @@ <h1>Current entities</h1>
 </dom-module>
 
 <script>
-class HaPanelDevState extends Polymer.Element {
+class HaPanelDevState extends window.hassMixins.EventsMixin(Polymer.Element) {
```

Comment:
<link rel='import' href='../../src/util/hass-mixins.html'>

Reply:
Just for my reference, what does the hass-mixins.html file do? Is it just adding support for older browsers that may not support all the features natively? I noticed it worked OK on my desktop with the latest Chrome without having that mixin file.

Reply:
these are HTML imports (in polymer 3 we will use es6 JavaScript modules instead). It works as the file was already included by another element